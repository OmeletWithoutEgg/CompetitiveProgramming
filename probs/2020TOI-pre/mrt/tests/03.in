41
13 10 3 5 3 8 7
3 2 0 2 0 1 0
5 1 1 0 1 2 4
13 9 8 4 11 3 6
3 2 1 2 1 1 1
13 10 3 5 3 4 4
13 1 4 0 2 8 11
2 1 1 0 1 1 1
5 1 4 1 4 3 2
5 3 2 1 1 2 4
3 2 1 0 2 2 1
13 3 11 9 0 11 6
7 5 0 4 0 2 2
3 2 0 2 0 1 0
5 0 1 4 0 2 2
17 8 8 7 4 11 13
2 1 1 0 1 1 1
13 1 6 11 10 4 3
7 5 1 0 1 5 2
11 8 3 6 5 2 7
13 3 0 1 11 8 9
7 6 2 2 1 1 3
3 1 1 1 0 0 1
7 2 0 5 3 6 2
2 1 1 1 1 0 1
11 8 1 8 7 7 9
2 0 1 0 1 1 0
2 1 1 1 1 0 1
11 1 1 3 8 0 1
11 6 3 9 4 4 6
5 4 4 0 3 4 3
11 1 9 6 0 8 1
3 1 2 2 0 1 2
2 0 1 0 1 1 0
2 0 1 1 0 0 1
17 0 10 14 0 5 13
11 3 7 8 2 9 2
3 2 1 1 1 2 1
11 10 4 0 2 6 7
7 3 1 4 0 6 5
7 6 0 5 6 6 0
