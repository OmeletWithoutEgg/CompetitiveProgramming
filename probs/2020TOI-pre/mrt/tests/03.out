9 4
0 0
4 2
5 3
0 1
5 3
2 1
1 1
2 3
1 1
2 1
2 5
2 2
0 0
0 4
3 12
1 1
12 11
4 6
6 5
7 0
1 3
0 1
4 4
1 1
0 1
0 1
1 1
0 1
3 2
1 3
0 8
1 2
0 1
0 1
11 10
5 2
0 1
6 7
5 6
4 5
